opusMT-servers-UH.json

```diff
@@ -1,83 +1,83 @@
 {
-    "192.168.1.32:25000" : {
-	"source-languages" : "en",
-	"target-languages" : "bcl"
-    },
-    "192.168.1.32:5001" : {
-	"source-languages" : "es",
-	"target-languages" : "aym+bzd+cni+gn+hch+nah+oto+quy+shp+tar"
-    },
-    "192.168.1.18:20000" : {
-	"source-languages" : "en",
-	"target-languages" : "fi"
-    },
-    "192.168.1.18:20001" : {
-	"source-languages" : "fi",
-	"target-languages" : "en"
-    },
-    "192.168.1.18:20005" : {
-	"source-languages" : "fi",
-	"target-languages" : "de"
-    },
-    "192.168.1.19:20006" : {
-	"model" : "goethe",
-	"source-languages" : "de",
-	"target-languages" : "fi"
-    },
-    "192.168.1.19:20004" : {
-	"source-languages" : "de",
-	"target-languages" : "fi"
-    },
-    "192.168.1.19:21000" : {
-    	"source-languages" : "et+fi",
-    	"target-languages" : "da+fo+is+no+nb+nn+sv"
-    },
-    "192.168.1.19:21001" : {
-    	"source-languages" : "da+fo+is+no+nb+nn+sv",
-    	"target-languages" : "et+fi"
-    },
-    "192.168.1.14:21100" : {
-	"source-languages" : "fr",
-	"target-languages" : "fi"
-    },
-    "192.168.1.14:21101" : {
-	"source-languages" : "fi",
-	"target-languages" : "fr"
-    },
-    "192.168.1.21:21050" : {
-    	"source-languages" : "ga+cy+br+gd+kw+gv",
-    	"target-languages" : "en"
-    },
-    "192.168.1.21:21051" : {
-	"source-languages" : "en",
-    	"target-languages" : "ga+cy+br+gd+kw+gv"
-    },
-    "192.168.1.21:22000" : {
-	"source-languages" : "se+sma+smj+smn+sms+vep+et+fi+kv+krl+nb+no+nn+ru+sv+en",
-    	"target-languages" : "se+sma+smj+smn+sms+vep+et+fi+kv+krl+nb+no+nn+ru+sv+en"
-    },    
-    "192.168.1.22:21004" : {
-    	"source-languages" : "de+af+fy+nl",
-    	"target-languages" : "de+af+fy+nl"
-    },
-    "192.168.1.22:21005" : {
-    	"source-languages" : "da+fo+is+no+nb+nn+sv",
-    	"target-languages" : "da+fo+is+no+nb+nn+sv"
-    },
-    "192.168.1.22:21006" : {
-    	"source-languages" : "et+hu+fi",
-    	"target-languages" : "et+hu+fi"
-    },
-    "192.168.1.15:20002" : {
-    	"source-languages" : "sv",
-    	"target-languages" : "fi"
-    },
-    "192.168.1.15:20003" : {
-    	"source-languages" : "fi",
-    	"target-languages" : "sv"
-    },
-    "192.168.1.15:21005" : {
-    	"source-languages" : "ca+es+fr+ga+it+la+oc+pt_br+pt",
-    	"target-languages" : "ca+es+fr+ga+it+la+oc+pt_br+pt"
+    "192.168.1.32:25000": {
+        "source-languages": "en",
+        "target-languages": "bcl"
+    },
+    "192.168.1.32:5001": {
+        "source-languages": "es",
+        "target-languages": "aym+bzd+cni+gn+hch+nah+oto+quy+shp+tar"
+    },
+    "192.168.1.18:20000": {
+        "source-languages": "en",
+        "target-languages": "fi"
+    },
+    "192.168.1.18:20001": {
+        "source-languages": "fi",
+        "target-languages": "en"
+    },
+    "192.168.1.18:20005": {
```

Comment:
Just curiosity, is there any reason to change this line (previously fi-de) and not any other?

Reply:
no XDDDD It was a random selection, not sure if this is needed

Reply:
This seems to point to a specific server instance in charge of that specific translation. But this is just a wild guessing, I haven't even read what the repo is about. I'd say there is no need.

I guess `services.json` defines what to instance and this file defines a list of external instances you cann connect to?

Reply:
Yes, I think services.json does what you say and it is enough to make it work